src/StatusQ/Controls/StatusSwitchTab.qml

```diff
@@ -0,0 +1,53 @@
+import QtQuick 2.14
+import QtQuick.Controls 2.14
+import QtGraphicalEffects 1.13
+
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+
+TabButton {
```

Comment:
if its based on `TabButton` its should be `StatusSwitchTabButton` bc inherits `AbstractButton` functionality. `Tab` sounds like a `Page`.

Reply:
Fair enough, will rename!